automated-deployments-using-github-actions.md

```diff
@@ -0,0 +1,400 @@
+
+> Replace the `GITHUB_USERNAME` in the URL below with your GitHub username to match the remote origin of the forked repository.
+
+    ```console
+       $ git clone https://github.com/{{GITHUB_USERNAME}}/okteto-flask-app
+    ```
+
+In the `flaskr.py` file, there are three API routes defined for performing a create, retrieve and delete operation against a connected [Couch database](https://couchdb.apache.org/).
+
+The cloned project also contains a `Dockerfile` and `docker-compose.yml` file that contains all defined steps and services needed to create a multi-stage build for this application.
+
+To run this application, execute the [docker-compose](https://docs.docker.com/compose/) command below to build and run the application container that consists of a `database` and `app` service.
+
+
+   ```console
+       $ docker-compose up --build
+    ```
+
+To test the application above, execute the command below from a new terminal window to make a POST request to the `/api/customer` api route within the flask API using [cURL](https://curl.se/) which inserts a new document into the customer collection within the running Couch database through it’s RESTful Apiserver.
+
+
+    ```console
+        $ curl -X POST -d '{"name":"Victory Nwani","occupation":"Software Engineer"}'  -H 'Content-Type: application/json' http://localhost:5050/api/customer
+     ```
+
+To view the data inserted from the `POST` request above, execute the command below from your terminal to make a GET request to the `/api/customer` api route within the flask API using [cURL](https://curl.se/) which retrieves all documents in the customer collection within the couch database.
+
+   ```console
+       $ curl http://localhost:5050/api/customer`
+    ```
+
+![GET request using curl from a terminal to fetch data from /api/customer endpoint](./flask_app_couch_data.png)
+
+
+You can also work with the running couch database through the [Fauxton web interface](https://couchdb.apache.org/fauxton-visual-guide/index.html) that comes by default with couchdb image at [http://localhost:5984/_utils](http://localhost:5984/_utils).
+
+> As defined in the environment field within the database service in the `docker-compose.yml` file, the couchdb username is **couchdb-admin**, while the password is **couchdb-password**. Feel free to change them to your own preferred secured values.
+
+
+## Step 2: Deploy Flask Application To Okteto Cloud
+
+Now you have the cloned application working locally on your computer. You will deploy the cloned version to the Okteto cloud to serve as your production deployment. The folder already contains a `Dockerfile` and a `docker-compose.yml` file that defines the services and steps needed to build the image for this application.
```

Comment:
So, Okteto isn't for production based deployment. Okteto is primarily for development so the theme for this article should be for development.

Reply:
Alright. 

I have applied this correction and adjusted the article's title.